designsafe/apps/api/filemeta/models.py

```diff
@@ -0,0 +1,17 @@
+from django.db import models
+from django.contrib.postgres.fields import JSONField
+
+
+class FileMetaModel(models.Model):
+    system = models.CharField(max_length=255)
+    path = models.CharField(max_length=4096)
+    data = models.JSONField()
```

Comment:
IMO the `data` field should be renamed to `value` since that's the naming convention in V2 metdata

Reply:
👍